cms/search/tests/test_resource_builders.py

```diff
@@ -250,3 +250,12 @@ def test_article_page_with_custom_title_negative(self):
         self.assertNotEqual(article_data["title"], self.article_page.display_title)
         # The title should match the full title
         self.assertEqual(article_data["title"], self.article_page.get_full_display_title())
+
+    def test_standard_information_page_with_old_url_path(self):
+        """build_resource_dict should include 'uri_old' if old_url_path is provided as a string."""
+        page = self.info_page
+        result = build_resource_dict(page, old_url_path="/home/info-page/")
+
+        self.assert_base_fields(result, page)
+        self.assertIn("uri_old", result)
+        self.assertEqual(result["uri_old"], "/info-page")  # old_url_path without the /home prefix
```

Comment:
Can we also have a negative test, to show that the result does _not_ contain `uri_old` when you _don't_ provide `old_url_path`?